contents/handbook/growth/sales/contracts.md

```diff
@@ -39,7 +39,16 @@ We use [PandaDoc](https://app.pandadoc.com/a/#/) to handle document generation,
    * **Client Address Information** - Needs to be their legal correspondence address (check with your customer contact)
    * **Client.Company** - The legal company name
    * **Contract.Discount** - The discount % (appears in the Additional credit purchase section)
-   * **Contract.EffectiveDate** - The start date of the contract expressed in the format `01 Feb 2023`
+   * **Contract.EffectiveDate**
+     - Set the start date of the contract in the format DD MMM YYYY (e.g., 01 Feb 2023). For a new customer, this would be the date they choose to start their subscription. For an existing customer, we have two options:
+       - **Immediate Activation:** If the customer wishes to start using the credits immediately, set the start date to the beginning of their current billing period. This backdating ensures that the credits are applied correctly to the current billing cycle.
+       - **Next Billing Cycle:** If the customer prefers to begin their annual plan at the start of their next billing cycle, set the start date accordingly. This option aligns the contract start date with the upcoming billing period.
```

Comment:
If they don't want to use the credits for their current bill cycle then the credits cannot be applied to their account until _after_ the current invoice is paid.

Reply:
clarified this in billing page!